kotest-framework/kotest-framework-plugin-gradle/build.gradle.kts

```diff
@@ -12,6 +12,8 @@ plugins {
    id("com.github.node-gradle.node") version "7.1.0"
 }
 
+System.setProperty("kotestVersion", Ci.publishVersion)
```

Comment:
I'm pretty sure this can be done properly in some other way, but I don't know my way around kotest well enough)